plutus-core/plutus-core.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:      3.0
 name:               plutus-core
-version:            1.1.1.0
+version:            1.1.1.1
```

Comment:
Please change this back. We cannot make a patch release from master. Also, we have a script for this and don't change the version number manually.

Reply:
> We cannot make a patch release from master.

Why not?

> we have a script for this and don't change the version number manually.

Cool. I'll let you take care of it then

Reply:
Our release process suggets tagging major/minor releases (x.y.z.0) from master, while patch releases (x.y.z.w) by backporting to x.y.z.w-1.

Reply:
There has been a lot of changes since 1.1.1.0, some of which are breaking, so if we make a release from master it should be at least 1.2.0.0.

Reply:
> There has been a lot of changes since 1.1.1.0,

That makes total sense. I am not sure about your dev/version/release process, but it might be a good idea to bump the version as soon as breaking change is introduced, this way it is always possible to release from master  and it is clear that current version is way ahead of the latest released one.

Feel free to close this PR. Revision takes care of this issue for me! Thank you

Reply:
Thanks for the suggestion. We are still experimenting on the release process.

By the way, we have a CI job that enforces adding to changelogs, and that's pretty useful - if the change logs are complete then at least it's easier to tell whether there are breaking changes when it's time to do a release.

Reply:
Ok, closing the PR as you suggested.